www/src/components/ecosystem/ecosystem-board.js

```diff
@@ -20,48 +24,92 @@ const EcosystemBoardRoot = styled("div")`
   }
 `
 
-const EcosystemBoard = ({
-  icons: { plugins: PluginsIcon, starters: StartersIcon },
-  starters,
-  plugins,
-  background,
-}) => {
-  return (
-    <EcosystemBoardRoot>
-      <EcosystemSection
-        title="Plugins"
-        description="With 429 plugins, lorem ipsum sunt we need some proper copy here pulling in data from your favorite source is only a few mouseclicks away."
-        subTitle="Featured Plugins"
-        icon={PluginsIcon}
-        links={[
-          { label: "Browse Plugins", to: "/plugins/" },
-          {
-            label: "Plugin Authoring",
-            to: "/docs/plugin-authoring/",
-            secondary: true,
-          },
-          { label: "Plugin Docs", to: "/docs/plugins/", secondary: true },
-        ]}
-        featuredItems={plugins}
-      />
-      <EcosystemSection
-        title="Starters"
-        description="Starters are boilerplate Gatsby sites maintained by the community which give you a headstart for your Gatsby project."
-        subTitle="Featured Starters"
-        icon={StartersIcon}
-        links={[
-          { label: "Browse Starters", to: "/starters/" },
-          { label: "Starter Docs", to: "/docs/starters/", secondary: true },
-        ]}
-        featuredItems={starters}
-      />
-      <EcosystemSection
-        title="External Resources"
-        description="More awesome Gatsby content, created by the community."
-        links={[{ label: "Browse Resources", to: "/docs/awesome-gatsby/" }]}
-      />
-    </EcosystemBoardRoot>
-  )
+class EcosystemBoard extends Component {
+  observer
+  observerTargets = []
+
+  componentDidMount() {
+    if (typeof window !== `undefined`) {
+      this.setupObserver()
+    }
+  }
+
+  componentWillUnmount() {
+    for (const target of this.observerTargets) {
+      this.observer.unobserve(target)
+    }
+  }
+
+  setupObserver = () => {
+    const options = { rootMargin: "0px", threshold: [1] }
+    this.observer = new IntersectionObserver(this.handleIntersect, options)
+    this.observerTargets = Array.from(
+      document.querySelectorAll(".featuredItems")
```

Comment:
Wish there were a better way to do this, but this seems _OK_!

Reply:
Potentially we could use `React.createRef` for here for `featuredItems` and pass it and use forwardRef down the road. Or move IntersectionObserver code to `ecosystem-featured-items.js`?

Reply:
Moving IntersectionOBserver to `ecosystem-featured-items.js` will result in two instances of it. Is not more efficient to have only one ?

Reply:
I'm not sure honestly. We do create new IntersectionObserver instance for each `gatsby-image` instance, so maybe this is not a problem? (Or is a problem and we should look to fix that in gatsby-image)

Reply:
Honestly, that's beyond my knowledge

Reply:
I think it (as is) is a reasonable way to handle this. If we have need to change it later, we can revisit.